vdf/__init__.py

```diff
@@ -86,10 +86,11 @@ def parse(fp, mapper=dict, merge_duplicate_keys=True, escaped=True):
     stack = [mapper()]
     expect_bracket = False
 
-    re_keyvalue = re.compile(r'^("(?P<qkey>(?:\\.|[^\\"])+)"|(?P<key>#?[a-z0-9\-\_\\\?]+))'
+    re_keyvalue = re.compile(r'^("(?P<qkey>(?:\\.|[^\\"])*)"|(?P<key>#?[a-z0-9\-\_\\\?$%<>]+))'
```

Comment:
Is the `qkey` from `+` to `*` intended? If I am reading this correctly that would make `""` (empty `str`) a valid key

Reply:
Yes, apparently .vmf files can contain empty keys. The issue was discovered in lasa01/io_import_vmf#47, the author sent me the .vmf that caused issues.

Reply:
I'm not surprised